CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# iwant-bot
+---
+## Hello everyone!
+- :star2: If you are reading this document then I'm assuming that you are interested in contributing to the __iwant-bot__ project.
+- This project is made by Kiwi.com interns. 
+- All contributions are welcome: new use-cases, code, patches, bugfixes, issues, etc. 
+- This is just a set of guidelines, not rules. Use your best judgement, and feel free to propose changes in a pull request.
+
+
+## Ways to contribute
+---
+  - At first, check existing [issues] and our [pull requests] and try to find something 
+  - [Create an issue], if you think you have a good idea and let us consider your contribution.
+  - To report a bug:
+      - Open an issue with the label "bug" :bug:.
+      - Describe expected behavior.
+      - Describe an actual, incorrect behavior.
+      - It would be great if a pull request with possible solution came along with the issue/the report.
+  - You can create your own [pull request](https://help.github.com/articles/creating-a-pull-request/) on [GitHub].
+
+### Tools integrated in CI and running for every commit
+---
+- Every commit is automatically tested on [circle CI] with tox and coala.
+- [coala] - linting and fixing code for all languages
+    - If you want to check you code before the commit, you could [install coala](http://docs.coala.io/en/latest/Users/Install.html) on your computer and check different enhancement proposals on your local machine.
+- [tox] - aims to automate and standardize testing in Python
+    - If you want to check you code before the commit, you could [install tox](https://tox.readthedocs.io/en/latest/install.html) on your computer and test your contribution on your local machine.
+    
+
+
+# Code contribution guide
+---
+- For every contribution make a pull request and make a comment on issue with your plan of implementing this issue.

```

Comment:
That should be understood as if you are creating a pull request and there is an related issue, then make a comment on the issue